### PR TITLE
Refactor buttons header

### PR DESCRIPTION
**Pull request by LeDeep**

### CODE REVIEW

File: client/containers/apply/license-and-id-history-form-container.jsx [hunk 1]
```diff
@@ -25,54 +25,54 @@ const ConnectedForm = (props) => {
...
-        />
+      <Page
+        sectionNumber='2'
+        sectionName='My history'
+        {...props}
+      >
+        <div>
+          <form onSubmit={onSubmit}>
+            <LicenseAndIdHistory
+              pageTitle      ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/license-and-id-history-form-container.jsx]: I am wondering about these `pageTitle` variables being needed in page partials. Do we still need these parts doing this work now that the page object is setting the title of the page? Or is the pageTitle a different kind of variable?

File: client/containers/apply/license-and-id-history-form-container.jsx [hunk 2]
```diff
@@ -25,54 +25,54 @@ const ConnectedForm = (props) => {
...
+      sectionName='My history'
+      {...props}
+    >
+      <div>
         <form onSubmit={onSubmit}>
           <LicenseAndIdHistory
-            pageTitle      ={pageTitle}
-            onChange       ={props.onChange}
-            selectedValue  ={props.licenseAndIdHistory.isIssued}
+            pageTitle     ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/license-and-id-history-form-container.jsx]: ?

File: client/containers/apply/license-issues-form-container.jsx [hunk 3]
```diff
@@ -25,52 +25,52 @@ const ConnectedForm = (props) => {
...
         <form onSubmit={onSubmit}>
           <LicenseIssues
-            pageTitle      ={pageTitle}
-            onChange       ={props.onChange}
-            selectedValue  ={props.licenseIssues.isSuspended}
-          />
-          <EnterRevokedSuspended
-            onChange       ={props.onChange}
-            licenseIssues  ={props.licenseIssues}
+            pageTitle     ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/license-issues-form-container.jsx]: ?

File: client/containers/apply/license-issues-form-container.jsx [hunk 4]
```diff
@@ -25,52 +25,52 @@ const ConnectedForm = (props) => {
...
-        />
+      <Page
+        sectionNumber='2'
+        sectionName='My history'
+        {...props}
+      >
+        <div>
+          <form onSubmit={onSubmit}>
+            <LicenseIssues
+              pageTitle      ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/license-issues-form-container.jsx]: ?

File: client/containers/apply/medical-history-form-container.jsx [hunk 5]
```diff
@@ -26,51 +25,53 @@ const ConnectedForm = (props) => {
...
+    <Page
+      sectionNumber='2'
+      sectionName='My history'
+      {...props}
+    >
+      <div>
+        <form onSubmit={onSubmit}>
+         <MedicalCondition
+            pageTitle      ={pageTitle}
+            onChange      = {props.onChange}
```
Comment (baccigalupi) [inline client/containers/apply/medical-history-form-container.jsx]: ?

File: client/containers/apply/medical-history-form-container.jsx [hunk 6]
```diff
@@ -26,51 +25,53 @@ const ConnectedForm = (props) => {
...
-    </div>
+    <Page
+      sectionNumber='2'
+      sectionName='My history'
+      {...props}
+    >
+      <div>
+        <form onSubmit={onSubmit}>
+          <MedicalCondition
+            pageTitle      ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/medical-history-form-container.jsx]: ?

File: client/containers/apply/names-history-form-container.jsx [hunk 7]
```diff
@@ -25,51 +25,53 @@ const ConnectedForm = (props) => {
...
-    </div>
+    <Page
+      sectionNumber='2'
+      sectionName='My history'
+      {...props}
+    >
+      <div>
+        <form onSubmit={onSubmit}>
+         <UsedPreviousNames
+            pageTitle      ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/names-history-form-container.jsx]: ?

File: client/containers/apply/names-history-form-container.jsx [hunk 8]
```diff
@@ -25,51 +25,53 @@ const ConnectedForm = (props) => {
...
-    </div>
+    <Page
+      sectionNumber='2'
+      sectionName='My history'
+      {...props}
+    >
+      <div>
+        <form onSubmit={onSubmit}>
+          <UsedPreviousNames
+            pageTitle      ={pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/names-history-form-container.jsx]: ?

File: client/containers/apply/organ-donation-form-container.jsx [hunk 9]
```diff
@@ -20,8 +20,6 @@ const ConnectedForm = (props) => {
 
   return (
     <div>
-      <HomeLink />
-
       <form onSubmit={onSubmit}>
```
Comment (baccigalupi) [inline client/containers/apply/organ-donation-form-container.jsx]: Does this need to get wrapped in a Page too? The HomeLink went away, but not sure what else is going on

File: client/containers/apply/physical-traits-form-container.jsx [hunk 10]
```diff
@@ -21,33 +21,36 @@ const ConnectedForm = (props) => {
...
-    </div>
+    <Page
+      sectionNumber='1'
+      sectionName='My basics'
+      {...props}
+    >
+      <div className="physical-traits-form">
+        <form onSubmit={onSubmit}>
+         <Sex
+            pageTitle     = {pageTitle}
```
Comment (baccigalupi) [inline client/containers/apply/physical-traits-form-container.jsx]: ?

File: client/presentations/voter/ballot-by-mail-form.jsx [hunk 11]
```diff
@@ -15,50 +15,49 @@ const FAQ_CLASSNAME_NO    = 'faq-ballot-by-mail-no';
...
 
-        <div className='inner-bottom'>
-          <SelectorCollection
-            name          = 'ballotByMail'
-            values        = {VALUES}
-            onChange      = {props.onChange}
-            selectedValue = {props.selectedValue}
-          />
-        </div>
+          <div className='inner-bottom'>
```
Comment (baccigalupi) [inline client/presentations/voter/ballot-by-mail-form.jsx]: The Yes/No state of the FAQ Drawer is new content? Or was it hiding somewhere else?

File: client/presentations/voter/voter-confirmation.jsx [hunk 12]
```diff
@@ -4,46 +4,46 @@ import React            from 'react';
...
-          linkText='Back' />
-
-        <div className='unit spacer' />
-
-        <LinkButton
-          to={linkAddress}
-          className='continue'
-          linkText='Continue' />
+    <Page
+      sectionNumber='3'
```
Comment (baccigalupi) [inline client/presentations/voter/voter-confirmation.jsx]: I think you might have to pass the pageTitle to the Page object, unless there is something else going on to get the pageTitle into the props.

Comment (baccigalupi) [reply client/presentations/voter/voter-confirmation.jsx]: That is probably true in other places where I overlooked it above

File: client/presentations/voter/voter-preferences-intro-form.jsx [hunk 13]
```diff
@@ -4,52 +4,51 @@ import React            from 'react';
...
-    <div>
-      <HomeLink />
-      <SectionHeader
-        number='3'
-        name='Voting registration'
-      />
+    <Page
+      sectionNumber='3'
+      sectionName='Voting registration'
+      {...props}
```
Comment (baccigalupi) [inline client/presentations/voter/voter-preferences-intro-form.jsx]: add page title?

File: client/presentations/voter/voter-preferences-intro-updated-form.jsx [hunk 14]
```diff
@@ -15,39 +15,39 @@ const VoterPreferencesIntroUpdated = (props) => {
   document.title = pageTitle;
```
Comment (baccigalupi) [inline client/presentations/voter/voter-preferences-intro-updated-form.jsx]: This can go now, and the pageTitle attribute should be passed in to the Page

**Comment by LeDeep** [comment]
@baccigalupi I passed pageTitle to the Page object in the presentation layer, and removed pageTitle from the containers. This is cleaner, and reduces some code repetition. Let me know what you think.

**Pull request merged**